discretize/View.py

```diff
@@ -646,7 +648,120 @@ def _plotCylTensorMesh(self, plotType, *args, **kwargs):
         return out
 
     def plotGrid(self, *args, **kwargs):
-        return self._plotCylTensorMesh('plotGrid', *args, **kwargs)
+        if self.isSymmetric:
+            return self._plotCylTensorMesh('plotGrid', *args, **kwargs)
+
+        # allow a slice to be provided for the mesh
+        slc = kwargs.pop('slice', None)
+        if isinstance(slc, str):
+            slc = slc.lower()
+        assert slc in ['theta', 'z', 'both', None], (
+            "slice must be either 'theta','z', or 'both' not {}".format(slc)
+        )
+
+        # if slc is None, provide slices in both the theta and z directions
+        if slc == 'theta':
+            self._plotGridThetaSlice(*args, **kwargs)
+        elif slc == 'z':
+            self._plotGridZSlice(*args, **kwargs)
+        else:
+            ax = kwargs.pop('ax', None)
+            if ax is not None:
+                if len(ax) != 2:
+                    warnings.warn(
+                        "two axes handles must be provided to plot both theta "
+                        "and z slices through the mesh. Over-writing the axes."
+                    )
+                    ax = None
+                else:
+                    # find the one with a polar projection and pass it to the
+                    # theta slice, other one to the z-slice
+                    polarax = [a for a in ax if a.__class__.__name__ == 'PolarAxesSubplot']
+                    if len(polarax) != 1:
+                        warnings.warn("""
+No polar axes provided. Over-writing the axes. If you prefer to create your
+own, please use
+
+    `ax = plt.subplot(121, projection='polar')`
+
+for reference, see: http://matplotlib.org/examples/pylab_examples/polar_demo.html
+                    https://github.com/matplotlib/matplotlib/issues/312
```

Comment:
it is a little annoying, the api for polar plots is more restricted. I have not found a way to do
```
fig, ax = plt.subplots(1, 2, figsize=(12, 4))
ax[0].make_this_polar
```